shard.lock

```diff
@@ -1,6 +1,6 @@
-version: 1.0
+version: 2.0
```

Comment:
Shard distributed with Crystal 0.33.0 doesn't seem to support this version. Could you revert this line, or update shard installed in GitHub Actions?

Reply:
Thank you!
I see. I re-sharded with crystal 0.33.0 and reverted it .